src/TapeChart.js

```diff
@@ -0,0 +1,48 @@
+class TapeChart {
+  constructor(users, rooms, bookings) {
```

Comment:
Looks like the class would need to be called by a promise all with a fetch for all 3 data bases.

Reply:
I'm planning to stick with one index.js file (and my class.js files of course) even though I'll have different html files.  

I am looking at how/where/when to instantiate next and don't feel confident quite yet.  
Also debating whether I actually need to pass the user data through here.
I think I will fetch straight away in my index.js file and use a promise all to instantiate this class.  I may refactor that later but its a good starting point for me.  

I'm also feeling very unclear on the best approach so I'm going for it with the knowledge that a fair amount of refactoring may be needed.